source/codegen/validate_linux_rt.py

```diff
@@ -39,16 +47,25 @@ def _need_linux_rt_feed_update(metadata_dir: str, changed_files: Set[str]) -> bo
     codegen_changes = _get_codegen_changes(changed_files)
     _print_changed_files("Codegen changes not affecting RT:", codegen_changes)
     remaining_changes = changed_files - codegen_changes
-    test_changes = _get_test_changes(changed_files)
+    test_changes = _get_test_changes(remaining_changes)
     _print_changed_files("Test related changes not affecting RT:", test_changes)
     remaining_changes -= test_changes
+    import_changes = _get_import_changes(remaining_changes)
+    _print_changed_files("Import related changes not affecting RT:", import_changes)
+    remaining_changes -= import_changes
+    markdown_changes = _get_markdown_changes(remaining_changes)
+    _print_changed_files("Markdown changes not affecting RT:", markdown_changes)
+    remaining_changes -= markdown_changes
 
     non_rt_drivers = [
         driver
         for driver in os.listdir(metadata_dir)
-        if not load_metadata(f"{metadata_dir}/{driver}/")["config"]["linux_rt_support"]
+        if os.path.isdir(os.path.join(metadata_dir, driver))
```

Comment:
This fixes it failing. It failed because it was trying to load_metadata for the "source/codegen/metadata/Imported_From_Hapigen.md" file which would fail. Want to restrict it to just check linux_rt_support for the metadata folders.